velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -175,6 +208,30 @@ velox::variant rowVariantAt(
   return velox::variant::row(std::move(values));
 }
 
+velox::variant mapVariantAt(
+    const ::duckdb::Value& vector,
+    const std::shared_ptr<const Type>& mapType) {
+  std::map<variant, variant> map;
```

Comment:
nit: std::unordered_map

Reply:
There is some subtle interplay between unordered_map and variant causing compilation issues. Leaving this as map for now.

Reply:
Wow! Would you share any details?

Reply:
In template: call to implicitly-deleted default constructor of 'std::hash<facebook::velox::variant>' error occurred here in instantiation of member function 'std::__unordered_map_hasher<facebook::velox::variant, std::__hash_value_type<facebook::velox::variant, facebook::velox::variant>, std::hash<facebook::velox::variant>, std::equal_to<facebook::velox::v... in instantiation of member function 'std::__hash_table<std::__hash_value_type<facebook::velox::variant, facebook::velox::variant>, std::__unordered_map_hasher<facebook::velox::variant, std::__hash_value_type<facebook::velox::variant, fa... in instantiation of member function 'std::unordered_map<facebook::velox::variant, facebook::velox::variant>::unordered_map' requested here '__enum_hash' has been explicitly marked deleted here